packages/main/src/Input.ts

```diff
@@ -1258,10 +1261,19 @@ class Input extends UI5Element implements SuggestionComponent, IFormInputElement
 			return;
 		}
 
-		const Suggestions = getComponentFeature<typeof InputSuggestions>("InputSuggestions");
-		if (Suggestions) {
+		const setup = (Suggestions: typeof InputSuggestions) => {
 			Suggestions.i18nBundle = Input.i18nBundle;
 			this.Suggestions = new Suggestions(this, "suggestionItems", true, false);
+		};
+
+		// If the features is preloaded (the user manually imported InputSuggestions.js), it is already available on the constructor
```

Comment:
I'll fix the typo (features -> feature) in another PR not to re-trigger the build just for that